matrix-sdk-crypto/src/main/java/org/matrix/androidsdk/crypto/interfaces/CryptoUtil.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.androidsdk.crypto.interfaces
+
+import android.content.Context
+import com.google.gson.Gson
+import com.google.gson.JsonElement
+import org.matrix.androidsdk.crypto.model.crypto.OlmPayloadContent
+import java.io.*
+import java.util.zip.GZIPOutputStream
+
+interface CryptoUtil {
```

Comment:
Is this really necessary? I think most of these methods should not come from outside but be hidden

Reply:
was the fastest way to provided facilities from matrix-sdk to matrix-sdk-crypto

Reply:
ok, I've removed this